File: CS/CustomizedTrackBar/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;
using DevExpress.XtraEditors.Drawing;

namespace CustomizedTrackBar
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void customTrackBar1_Properties_DrawingTick(object sender, EventArgs e)
        {
            CustomizedTrackBar.DrawingTickEventArgs ee = e as DrawingTickEventArgs;
            TrackBarObjectInfoArgs tbObject = ee.TrackBarObject;
            CustomizedTrackBar.CustomTrackBarViewInfo vi = tbObject.ViewInfo as CustomizedTrackBar.CustomTrackBarViewInfo;
            vi.RepositoryItem.TickDisplayText = ee.TickCount.ToString();
        }

        private void customRangeTrackBar1_Properties_DrawingTick(object sender, EventArgs e)
        {
            CustomizedRangeTrackBar.DrawingTickEventArgs ee = e as CustomizedRangeTrackBar.DrawingTickEventArgs;
            TrackBarObjectInfoArgs tbObject = ee.TrackBarObject;
            CustomizedRangeTrackBar.CustomRangeTrackBarViewInfo vi = tbObject.ViewInfo as CustomizedRangeTrackBar.CustomRangeTrackBarViewInfo;
            vi.RepositoryItem.TickDisplayText =  Math.Pow(ee.TickCount, 2).ToString();
        }
    }
}

File: VB/CustomizedTrackBar/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Linq
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.XtraEditors.Drawing

Namespace CustomizedTrackBar
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

        Private Sub customTrackBar1_Properties_DrawingTick(ByVal sender As Object, ByVal e As EventArgs) Handles customTrackBar1.Properties.DrawingTickEvent
            Dim ee As CustomizedTrackBar.DrawingTickEventArgs = TryCast(e, DrawingTickEventArgs)
            Dim tbObject As TrackBarObjectInfoArgs = ee.TrackBarObject
            Dim vi As CustomizedTrackBar.CustomTrackBarViewInfo = TryCast(tbObject.ViewInfo, CustomizedTrackBar.CustomTrackBarViewInfo)
            vi.RepositoryItem.TickDisplayText = ee.TickCount.ToString()
        End Sub

        Private Sub customRangeTrackBar1_Properties_DrawingTick(ByVal sender As Object, ByVal e As EventArgs) Handles customRangeTrackBar1.Properties.DrawingTickEvent
            Dim ee As CustomizedRangeTrackBar.DrawingTickEventArgs = TryCast(e, CustomizedRangeTrackBar.DrawingTickEventArgs)
            Dim tbObject As TrackBarObjectInfoArgs = ee.TrackBarObject
            Dim vi As CustomizedRangeTrackBar.CustomRangeTrackBarViewInfo = TryCast(tbObject.ViewInfo, CustomizedRangeTrackBar.CustomRangeTrackBarViewInfo)
            vi.RepositoryItem.TickDisplayText = Math.Pow(ee.TickCount, 2).ToString()
        End Sub
	End Class
End Namespace
